locale/fa_IR/emails.po

```diff
@@ -10,6 +10,874 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"POT-Creation-Date: 2019-12-02T16:17:19-08:00\n"
+"PO-Revision-Date: 2019-12-02T16:17:19-08:00\n"
+"Language: \n"
+
+msgid "emails.passwordResetConfirm.subject"
```

Comment:
Likewise, not sure all these changes are intentional.

Reply:
Fixed.